Dockerfile

```diff
@@ -27,7 +16,9 @@ EXPOSE 139 445
 
 COPY . /container/
 
-HEALTHCHECK CMD ["/container/scripts/docker-healthcheck.sh"]
+HEALTHCHECK --interval=60s --timeout=15s \
+ CMD smbclient -L \\localhost -U % -m SMB3
+
```

Comment:
@MarvAmBass, you might want to update the healthcheck to `smbclient -L //localhost -U %`. I suggest to remove `-m SMB3`, as the user might disable SMBv3.

Reply:
thanks, but I didn't use smbclient for healthchecking

Reply:
True, I just wanted to suggest to use it. YMMV :wink:

Reply:
thanks, it thought the same, that it's a bit better than my current healthcheck - I'll think about it :)